learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: ballerina-language-basics-left-nav-pages-swanlake
+title: Writing a gRPC Service with Ballerina
+description: This guide will walk you through writing a simple Ballerina gRPC service and invoking the service through Ballerina gRPC client application.
+keywords: ballerina, grpc, protocol buffers, protobuf, ballerina packages, language-guide, standard library
+permalink: /learn/user-guides/getting-started/writing-a-grpc-service-with-ballerina
+active: language-basics
+intro: This guide will walk you through writing a simple Ballerina gRPC service and invoking the service through Ballerina gRPC client application.
+redirect_from:
+- /learn/getting-started/writing-a-grpc-service-with-ballerina
+- /learn/getting-started/writing-a-grpc-service-with-ballerina/
+- /learn/user-guide/getting-started/writing-a-grpc-service-with-ballerina
+- /learn/user-guide/getting-started/writing-a-grpc-service-with-ballerina/
+---
+
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Prerequisites](#prerequisites)
+- [Creating Service Definition](#creating-service-definition)
+- [Implementation](#create-a-ballerina-project)
+    - [Developing a gRPC service](#developing-a-gRPC-service)
+       - [Create a Ballerina project](#create-a-ballerina-project)
+       - [Generate source code](#generate-source-code)
+       - [Update and run the application](#update-and-run-the-application)
+    - [Developing a gRPC client](#developing-a-gRPC-client)
+        - [Create a Ballerina project](#create-a-ballerina-project)
+        - [Generate source code](#generate-source-code)
+        - [Update and run the application](#update-and-run-the-application)
+
+## Prerequisites
+To complete this tutorial, you need:
+
+- A command terminal
```

Comment:
```suggestion
- A command terminal
```
This should be consistent with the GraphQL guide. Refer the review comments in https://github.com/ballerina-platform/ballerina-dev-website/pull/3369.